addons/calendar/static/src/js/calendar_provider_sync_wizard_form_view.js

```diff
@@ -0,0 +1,13 @@
+/** @odoo-module **/
```

Comment:
apparently the views / renderer / controller files are now stored under a "/static/src/views" folder (new owl guidelines)